### DISCUSSION TITLE
Pandas Visualisation of the response

### DISCUSSION THREAD

**Discussion by Saurav-Navdhare** [sinaptik-ai/pandas-ai#220] [Q&A]
Many times from the LLM we receive a table like structure, do Pandas AI reforms that output and make a Pandas DF?

**Comment by gventuri** [comment]
Hi @Saurav-Navdhare, this is not happening yet. You suggest we support a new feature that converts the tables generated by LLMs and convert them to dataframes?

**Reply by Saurav-Navdhare** [reply]
Yes, It would be great if the table output is transformed into a dataframe, else the output provided in other format from which we can generate a dataframe.
